mne_qt_browser/tests/test_pg_specific.py

```diff
@@ -4,6 +4,8 @@
 # License: BSD-3-Clause
 
 import numpy as np
+import math
```

Comment:
```suggestion
import math
import numpy as np
```

import standard lib first

Reply:
with later changes math lib no longer needed.